.circleci/config.yml

```diff
@@ -34,7 +34,19 @@ jobs:
       - deploy:
           name: Deploy
           command: |
+            # Do not release on Broad Holidays
+            CURRDATE=$(date +%m/%d/%Y)
+             array=("03/12/2021" "4/19/2021" "5/28/2021" "5/31/2021" "7/2/2021" "7/5/2021"
+             "9/6/2021" "10/11/2021" "11/11/2021" "11/25/2021" "11/26/2021" "12/23/2021" "12/24/2021"
+              "12/27/2021" "12/28/2021" "12/29/2021" "12/30/2021" "12/30/2021")
```

Comment:
wow, thanks for making this change Emily!

one question I have is is it possible to make it not care about year? cuz otherwise we'll need to update this every year

Reply:
I went with the yearly approach thinking the majority of the dates would be different in a given year but maybe it would be worthwhile to separate into a "yearly holidays" list that reoccur every year on the same date and a "year specific" list for the unique ones so less has to be changed each time.

Reply:
Alright I separated the Holidays that are the same date yearly, please let me know if that fits more with what you were thinking!